examples/all-clusters-app/ambd/main/Makefile

```diff
@@ -0,0 +1,207 @@
+#
```

Comment:
What is the common opinion on the use of makefile ? We all migrate to GN or cmake in the beginning of Matter.

Reply:
I don't think it's a good idea to use make here. The platform SDK already using it is one thing, but why add a brand new example using it?